cpp/cmake/modules/ConfigureCUDA.cmake

```diff
@@ -34,9 +34,13 @@ if(DISABLE_DEPRECATION_WARNING)
 endif()
 
 # Option to enable line info in CUDA device compilation to allow introspection when profiling / memchecking
-if(CUDA_ENABLE_LINEINFO)
-    list(APPEND CUML_CUDA_FLAGS -lineinfo)
-endif()
+if(CUDA_ENABLE_LINE_INFO)
+  list(APPEND CUML_CUDA_FLAGS -lineinfo)
+endif(LINE_INFO)
```

Comment:
CMake emits a warning if the names don't match between if and endif.

Reply:
That is no longer the case as of a recent CMake version if I’m not mistaken, with it being marked as legacy in CMake’s [docs](https://cmake.org/cmake/help/latest/command/if.html#synopsis): 

```
Per legacy, the else() and endif() commands admit an optional <condition> argument. If used, it must be a verbatim repeat of the argument of the opening if command.
```

And as of CMake 3.20+ (the required version), I don’t see any warnings

Reply:
Thanks for the references!